### PR TITLE
Albert Jamie run quiz view

### PR DESCRIPTION
**Pull request by JamieJChew**
These changes should allow the user to fully quiz themselves on a quiz of their choice. The screens have not been formatted or anything yet, but everything is functional. 

These changes specifically were made to fix a bug where when quiz ended, the user was brought back to a deck screen rather than the quiz screen.

### CODE REVIEW

File: src/main/java/runQuiz/RunQuizPresenter.java [hunk 1]
```diff
@@ -60,8 +60,6 @@ public void prepareSuccessView(ShowProblemOutputData outputData) {
             viewBound.updateView(viewModel);
         }
 
-        System.out.println("Not sure how we got here");
```
Comment (AndrewNQN) [inline src/main/java/runQuiz/RunQuizPresenter.java:63]: Thanks for removing this. Albert just did not know what was happening

File: src/main/java/screens/View.java [hunk 2]
```diff
@@ -222,13 +222,13 @@ private void menuSwitch() {
...
                 //Menu that displays the quizzes.
-
+                quizScreen.setVisible(true);
 
                 break;
 
             case SHOW_ANSWER:
                 //Menu that shows the answer to a flashcard along with a show next problem button.
-                showAnswerScreen.setQuiz(cardIdArray, currCardIndex, cardAnswer);
+                showAnswerScreen.setQuiz(cardIdArray, currCardIndex, cardAnswer, userAnswer);
```
Comment (AndrewNQN) [inline src/main/java/screens/View.java:231]: If im understanding this right, both the user answer and card answer should now both be shown on the screen (after a creating a JLabel)?

**Review by AndrewNQN — APPROVED** [review]
Good changes. I will make my pull request after I merge these changes on.

File: src/main/java/runQuiz/RunQuizPresenter.java (hunk 1 above)
Comment (JamieJChew) [reply src/main/java/runQuiz/RunQuizPresenter.java:63]: Oh yeah, sorry about that, that was a remnant of back when there were return statements above that line.

**Review by AndrewNQN — APPROVED** [review]
From what I can tell, options are now also being sent from the presenter so the user can see all the options for a multiple choice card when quizzing. Much needed change. All other changes have been addressed in my previous comments.

**Review by jakan9 — APPROVED** [review]
This should hopefully solve the bug that sent the user to the deck screen. It is important for the user to review their own answer as well so this helps in that as well.

**Pull request merged**